src/recording/recordingmanager.cpp

```diff
@@ -145,23 +167,37 @@ QString& RecordingManager::getRecordingDir() {
 }
 
 // Only called when recording is active.
-void RecordingManager::slotDurationRecorded(QString durationStr)
+void RecordingManager::slotDurationRecorded(quint64 duration)
```

Comment:
We have a nice class mixxx::Duration with 64 bit nano seconds resolution that includes some convenient helpers for printing and debugging. Maybe it is worth to use this here.